src/utilities/music/MusicManager.java

```diff
@@ -1,8 +1,12 @@
 package utilities.music;
 
+import java.net.URI;
+import java.net.URISyntaxException;
 import java.util.HashMap;
 import java.util.Map;
 
+import com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeAudioSourceManager;
+import com.sedmelluq.discord.lavaplayer.source.youtube.YoutubeSearchProvider;
```

Comment:
Is there any reason why these imports are here without being used in `MusicManager`?

Reply:
The reason is that i forgot to remove them